message.go

```diff
@@ -113,10 +113,18 @@ func (message Message) QuitSent() bool {
 	return message.quitSent
 }
 
-// Message consistency status predicate. Returns true for case when message struct is consistent.
-// It means that MAILFROM, RCPTTO, DATA commands and message context were successful.
-// Otherwise returns false
-func (message Message) isConsistent() bool {
+// Getter for message consistency status predicate. Returns true
+// for case when message struct is consistent. It means that
+// MAILFROM, RCPTTO, DATA commands and message context
+// were successful. Otherwise returns false
+func (message Message) IsConsistent() bool {
+	return message.mailfrom && message.rcptto && message.data && message.msg
+}
+
+// Message pointer consistency status predicate. Returns true for case
+// when message struct is consistent. It means that MAILFROM, RCPTTO, DATA
+// commands and message context were successful. Otherwise returns false
+func (message *Message) isConsistent() bool {
```

Comment:
Right, I see where this must be preserved on the pointer type, for sure. :+1:

Reply:
Yeah 👨‍💻

Reply:
I'm going to create an issue to add more test coverage to `handler` now, I will have some time over the weekend to do it and doing so will help me understand that part of the code better. I'm really glad to be invited to participate more on this project, Vladislav, thanks!